lib/src/entities/user_profile.dart

```diff
@@ -27,6 +27,10 @@ class UserProfile with _$UserProfile {
     @JsonKey(name: 'userBioData') BioData? userBioData,
     @JsonKey(name: 'homeAddress') Address? homeAddress,
     @JsonKey(name: 'workAddress') Address? workAddress,
+
+    /// [assistant] refers to the name of the Be.Well user assistant,
+    /// i.e Bev or Bowi.
+    @JsonKey(name: 'assistant') String? assistant,
```

Comment:
An enum will be great here instead of string. With enum you can do validation with ease.

Reply:
Okay. Looking into it.

Reply:
Fixed